# request constants
uint8 VELOCITY_OBSTACLE=0
uint8 EXTERNAL=1
uint8 BEHIND_ROBOT=2
uint8 PREDICTED_GOAL=3
# request fields
uint8                                     type
float64[]                                 predict_times
int64[]                                   ids
---
# response fields
agent_path_prediction/PredictedPoses[]    predicted_agents_poses
